lib/gruff/base.rb

```diff
@@ -166,7 +166,7 @@ class Base
     #
     # @param target_width [Numeric, String] The graph image width.
     #
-    def initialize(target_width = DEFAULT_TARGET_WIDTH)
+    def initialize(target_width = DEFAULT_TARGET_WIDTH, delimiter: ',', separator: '.')
```

Comment:
If you will want to add more arguments, I think they would only settings for the canvas in here.

Reply:
Please add comments according to the yard notation if you change something in public API.

Reply:
I will add an attribute or a method to set it if needed

Reply:
Thanks for the feedback! I'll check it out soon-ish.